source/shared/net/guides/add-baker-desktop.rst

```diff
@@ -31,15 +31,45 @@ There are a couple of things you must set up in preparation before you can start
 - Set up a new account that you'll be using as baker account.
 - Verify that the account balance has the required amount of CCD.
 
+Add baker (Single-signature account)
+====================================
+
+#. Go to **Accounts**, select the account you want to add as baker account, change the view to the detailed view, and select **Add baker**.
+
+#. Specify the amount that you want to stake where it says **Amount**. The more you stake the greater is the probability that your account will be chosen to bake the next block.
+
+   Baker accounts receive a reward when they have baked a block, and the reward
+   is added to the staked amount on the account by default. However, you can change this setting so that the reward is added to the disposable amount instead.
+
+   -  Select **No, don’t restake** if you'd rather add the rewards to the disposable amount on the account.
+
+#. You have to export the baker credentials so that you can start the node with the baker keys. Select **Export baker credentials** and navigate to the place on your computer where you want to save the file. If you're on Windows make sure that **All Files** is selected in **Save as type**. Give the file a name and the extension .json, and then click **Save** and navigate to the place on your computer where you want to save the file.
```

Comment:
```suggestion
#. You have to export the baker credentials so that you can start the node with the baker keys. Select **Export baker credentials** and navigate to the place on your computer where you want to save the file. If you're running Windows make sure that **All Files** is selected in **Save as type**. Give the file a name and the extension .json, and then click **Save** and navigate to the place on your computer where you want to save the file.
```